cmd/deploy/command_annotations.go

```diff
@@ -0,0 +1,19 @@
+package deploy
+
+import "github.com/spf13/cobra"
+
+func storeCommandResult(cmd *cobra.Command, key, value string) {
+
+	if cmd.Annotations == nil {
+		cmd.Annotations = make(map[string]string)
+	}
+
+	cmd.Annotations[key] = value
+}
+
+func fetchCommandResult(cmd *cobra.Command, key string) string {
+	if cmd.Annotations == nil {
+		return ""
+	}
+	return cmd.Annotations[key]
```

Comment:
It took me a few minutes to understand this use of the `Annotations`. Would you please add a comment describing what its for? Sadly the description from cobra didn't help me:
> // Annotations are key/value pairs that can be used by applications to identify or group commands.